.ai/migrations/config.yml

```diff
@@ -0,0 +1,487 @@
+# SQL 句型启用禁用清单
```

Comment:
This file content is duplicated. Please use a soft link to `.erda/migrations/config.yml`.